.github/workflows/build.yaml

```diff
@@ -65,10 +59,10 @@ jobs:
     needs:
       - bump_version
     steps:
-      - name: Checkout tagged branch of jade-data-repo
-        uses: actions/checkout@v3
+      - name: Checkout jade-data-repo
+        uses: actions/checkout@v4
         with:
-          ref: ${{ needs.bump_version.outputs.api_image_tag }}
+          # ref: ${{ needs.bump_version.outputs.tag }}
```

Comment:
Is `ref` no longer needed here?

Reply:
Correct, and I should've deleted the commented out line. The reason is that using `actions/checkout` without specifying a ref will "default to the reference or SHA for that event. Otherwise, uses the default branch" ([reference](https://github.com/actions/checkout?tab=readme-ov-file#usage)). This is the behavior we want since this workflow will now run on PRs as well as on merge to the default branch `develop`, so the current code should be pulled instead of always using `develop`.